bootstrap.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-set -u
-set -e
+# Set failure modes. Details: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
```

Comment:
Thanks for including this link. I learned a few things. I also love the fact that we can refer to a post from 2015 and it's still valid today. Yay for stable software environments (#D47G)